docs/dev/guide.rst

```diff
@@ -744,18 +746,12 @@ When calling the file directly, only the tests in this file for the bot will be
 
 See the :ref:`testing` section about how to run the tests.
 
-Configuration
-=============
-
-In the end, the new information about the new bot should be added to BOTS file
-located at `intelmq/bots`. Note that the file is sorted!
-
 Cache
 =====
 
 Bots can use a Redis database as cache instance. Use the `intelmq.lib.utils.Cache` class to set this up and/or look at existing bots, like the `cymru_whois` expert how the cache can be used.
 Bots must set a TTL for all keys that are cached to avoid caches growing endless over time.
-Bots must use the Redis databases `>=` 10, but not those already used by other bots. See `bots/BOTS` what databases are already used.
+Bots must use the Redis databases `>=` 10, but not those already used by other bots. Look at `intelmqctl list bots` to see which databases are already used.
```

Comment:
This command does not show the databases. Maybe recommend `find intelmq -type f -name '*.py' -exec grep -r 'redis_cache_db' {} \+` (not good, but it works).

Reply:
I've used the proposed command. Its the developer docs, so I think its okey for now.